spinnaker-jenkins-to-vmss/scripts/set_azure_spinnaker.sh

```diff
@@ -109,9 +109,12 @@ sudo printf "rosco.yml file has been updated\n"
 sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys EB9B1D8886F44E2A
 sudo printf "apt-key done\n" 
 
+# Enable Azure provider in /etc/default/spinnaker
+sudo sed -i '/SPINNAKER_AZURE_ENABLED=false/s/.*/SPINNAKER_AZURE_ENABLED=true/' /etc/default/spinnaker
```

Comment:
We pass in "--cloud_provider azure" to InstallSpinnaker.sh and this already gets set [here](https://github.com/spinnaker/spinnaker/blob/master/InstallSpinnaker.sh#L734)